ui/raidboss/data/05-shb/raid/e9n.js

```diff
@@ -1,8 +1,153 @@
+    },
+    {
+      id: 'E9N The Art Of Darkness Left',
+      netRegex: NetRegexes.startsUsing({ id: '5224', source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 4,
+      response: Responses.goRight(),
+    },
+    {
+      id: 'E9N Wide-Angle Particle Beam',
+      netRegex: NetRegexes.startsUsing({ id: '5AFF', source: 'Cloud Of Darkness', capture: false }),
+      response: Responses.awayFromFront('info'),
+    },
+    {
+      id: 'E9N Zero-Form Particle Beam',
+      netRegex: NetRegexes.startsUsing({ id: '55EB', source: 'Cloud Of Darkness' }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'E9N Empty Plane',
+      netRegex: NetRegexes.startsUsing({ id: '4FC6', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E9N Mire Of Despair',
+      netRegex: NetRegexes.startsUsing({ id: '5B07', source: 'Cloud Of Darkness', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Out from Under',
+        },
+      },
+    },
+    {
+      id: 'E9N Obscure Woods',
+      netRegex: NetRegexes.startsUsing({ id: '4FA2', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E9N Waste Away',
+      netRegex: NetRegexes.startsUsing({ id: '55DE', source: 'Cloud Of Darkness', capture: false }),
+      response: Responses.knockback('info'),
+    },
+    {
+      id: 'E9N Stygian Tether',
+      netRegex: NetRegexes.headMarker({ id: '000C' }),
+      condition: Conditions.targetIsYou(),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Wait Near Bramble',
+        },
+      },
+    },
+    {
+      id: 'E9N Stygian Break Tether',
+      netRegex: NetRegexes.headMarker({ id: '000C' }),
+      condition: Conditions.targetIsYou(),
+      delaySeconds: 3,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Away From Bramble',
+        },
+      },
+    },
+    {
+      id: 'E9N Wide-Angle Phaser',
+      netRegex: NetRegexes.startsUsing({ id: ['55DF', '55E[01]'], source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 5,
+      suppressSeconds: 1,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Go Sides At Tethered Wall',
+        },
+      },
+    },
+    {
+      id: 'E9N Rejuvenating Balm',
+      netRegex: NetRegexes.startsUsing({ id: '55E2', source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 5,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Away From Tethered Walls',
```

Comment:
Is it possible to make this "go N/S" or "go E/W"? Or, am I overthinking and this is sufficient?

Reply:
This is sufficient, but we can certainly try to add that; the Stygian Arbors are their own mobs with cast bars and positions, so you could certainly figure it out. But, it is very forgiving if you're anywhere aside from the absolute worst position.